CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - [cygnus-ngsi][feature] Add specific data truncation implementation for NGSICKANSink (#1344)
 - [cygnus] Add Travis CI (#1347)
 - [cygnus-ngsi][hardening] Force lowercase organization, dataset and resource names in NGSICKANSink (#1352)
+- [cygnus-ngsi][hardening] Replace warnings upon unnecessary herader received with debug traces at NGSIRestHandler (#1354)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1355/commits/c8e0dd219c38e492f683bfac50a23b707c5c2a19